tests/phpunit/tests/Checker/Default_Check_Collection_Tests.php

```diff
@@ -32,22 +32,24 @@ public function set_up() {
 	}
 
 	public function test_to_array() {
-		$this->assertSame(
-			array_values( $this->checks ),
-			$this->collection->to_array()
-		);
+		$collection_array = $this->collection->to_array();
+
+		$this->assertContains( $this->checks['static_check'], $collection_array );
+		$this->assertContains( $this->checks['runtime_check'], $collection_array );
```

Comment:
All the changes in these tests are fine, but not really great, as they make the tests more complicated. I realize this is needed to accommodate the change that `Default_Check_Repository` now includes all the default checks, but there's a cleaner way.

I think eventually we should decouple the default logic for registering checks from the actual list of default checks. Maybe we can introduce a `Empty_Check_Repository` class that will take over everything from `Default_Check_Repository`, except the default checks registration. Then `Default_Check_Repository` can be changed to extend that class with the only difference being the check registration.

The tests here could then all be kept as is, simply by using `Empty_Check_Repository` instead of `Default_Check_Repository`. That would still cover all the necessary logic.